lib/ui/painting/paint.cc

```diff
@@ -71,14 +71,8 @@ DlColor ReadColor(const tonic::DartByteData& byte_data) {
   // Invert alpha so 0 initialized buffer has default value;
   float alpha = 1.f - float_data[kColorAlphaIndex];
   uint32_t colorspace = uint_data[kColorSpaceIndex];
-  (void)colorspace;
-  uint32_t encoded_color =
-      static_cast<uint8_t>(std::round(alpha * 255.f)) << 24 |  //
-      static_cast<uint8_t>(std::round(red * 255.f)) << 16 |    //
-      static_cast<uint8_t>(std::round(green * 255.f)) << 8 |   //
-      static_cast<uint8_t>(std::round(blue * 255.f)) << 0;
-  // TODO(gaaclarke): Pass down color info to DlColor.
-  return DlColor(encoded_color);
+  return DlColor(alpha, red, green, blue,
```

Comment:
Shower thought, don't we need to do any color space conversions to RGBA/extend RGBA here? Or should we do that in dispatch based on the destination surface?

Reply:
The last PR added `DlColor::withColorSpace` for this conversion.  The next PR is an integration test that makes sure everything works.  It's been a couple of weeks since I tested the whole flow.  It's possible it got lost in all this juggling.  Let me double check I didn't lose it, because I'm not seeing a call to `DlColor::withColorSpace` anymore.

Reply:
You are correct, sir, reintroduced.  When's that monorepo coming again?